org.onionshare.OnionShare.yaml

```diff
@@ -2,11 +2,11 @@
 app-id: org.onionshare.OnionShare
 command: onionshare
 runtime: org.kde.Platform
-runtime-version: "5.15-21.08"
+runtime-version: "6.5"
```

Comment:
why 6.5?

1. latest for Qt6 is 6.6
2. latest for Qt5 is 5.15-23.08

the latter is probably what you want.

Reply:
thanks, Qt5 is probably what we should use? There need to be some adaptions to work on Qt6 probably, right?